suites/squid/nvmeof/tier-1_nvmeof_gwgroup_sanity.yaml

```diff
@@ -0,0 +1,229 @@
+      name: deploy NVMeoF services on 4 GW groups having 8 GW nodes
+
+  - test:
+      abort-on-fail: true
+      config:
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        cleanup:
+          - pool
+          - gateway
+          - initiators
+        gw_groups: # Configure gatewayGroups
+          - gw_group: group1
+            gw_nodes:
+              - node6
+              - node7
+            subsystems: # Configure subsystems with all sub-entities
+              - nqn: nqn.2016-06.io.spdk:cnode10
+                listener_port: 4420
+                listeners: [node6, node7]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+              - nqn: nqn.2016-06.io.spdk:cnode2
+                listener_port: 4420
+                listeners: [node6, node7]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+          - gw_group: group2
+            gw_nodes:
+              - node8
+              - node9
+            subsystems:
+              - nqn: nqn.2016-06.io.spdk:cnode3
+                listener_port: 4420
+                listeners: [node8, node9]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+              - nqn: nqn.2016-06.io.spdk:cnode4
+                listener_port: 4420
+                listeners: [node8, node9]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+              - nqn: nqn.2016-06.io.spdk:cnode5
+                listener_port: 4420
+                listeners: [node8, node9]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+          - gw_group: group3
+            gw_nodes:
+              - node10
+              - node11
+            subsystems:
+              - nqn: nqn.2016-06.io.spdk:cnode6
+                listener_port: 4420
+                listeners: [node10, node11]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+          - gw_group: group4
+            gw_nodes:
+              - node12
+              - node13
+            subsystems:
+              - nqn: nqn.2016-06.io.spdk:cnode7
+                listener_port: 4420
+                listeners: [node12, node13]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+              - nqn: nqn.2016-06.io.spdk:cnode8
+                listener_port: 4420
+                listeners: [node12, node13]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+        initiators:                             # Configure Initiators with all pre-req
```

Comment:
This attribute key is not used anywhere.

Reply:
Its being used in run.py [here](https://github.com/red-hat-storage/cephci/blob/c0b40dc2e0beae08a32cca69579c62c45e8092ee/tests/nvmeof/test_nvmeof_gwgroup_sanity.py#L128)